sphinx-docs/Server-Configuration.md

```diff
@@ -0,0 +1,108 @@
+# Server Configuration
+
+## Startup parameters
+
+`server.py` supports the following arguments:
+
+- `--log {DEBUG,INFO,WARNING,ERROR,CRITICAL}`: Sets the log option. The `DEBUG` option is useful for troubleshooting.
+- `--fresh`: Deletes all non-plugin data including custom abilities and adversaries, operations, and the agent list.
+- `--environment ENVIRONMENT`: Sets a custom configuration file. See "Custom configuration files" below for additional details.
+- `--plugins PLUGINS`: Sets CALDERA to run only with the specified plugins
+- `--insecure`: Uses the `conf/default.yml` file for configuration, not recommended.
+
+## Configuration file
+
+Caldera's configuration file is located at `conf/local.yml`, written on the first run. If the server is run with the `--insecure` option (not recommended), CALDERA will use the file located at `conf/default.yml`.
+
+Configuration file changes must be made while the server is shut down. Any changes made to the configuration file while the server is running will be overwritten.
+
+The YAML configuration file contains all the configuration variables CALDERA requires to boot up and run. A documented configuration file is below:
+
+```yaml
+ability_refresh: 60  # Interval at which ability YAML files will refresh from disk 
+api_key_blue: BLUEADMIN123  # API key which grants access to CALDERA blue
+api_key_red: ADMIN123  # API key which grants access to CALDERA red
+app.contact.dns.domain: mycaldera.caldera  # Domain for the DNS contact server
```

Comment:
It looks like some of the DNS configuration info may have gotten lost during the merge conflict resolution. Is it worth including some of the info from the original file? https://github.com/mitre/fieldmanual/pull/64/files#diff-8d8033dc12d18ad455b4300a0f7237f86700a4f8fd177e0cbd566b064272fc51L45-L46

Reply:
The only information I see missing is "The C2 server will respond with the NXDOMAIN response code for any other domain". It feels more appropriate to put this into a page or section dedicated to the different contact methods.